cluster/down.go

```diff
@@ -0,0 +1,43 @@
+package cluster
+
+import (
+	"github.com/rancher/rke/hosts"
+	"github.com/rancher/rke/services"
+)
+
+func (c *Cluster) ClusterDown() error {
```

Comment:
We probably should tear down stuff in the revers order they were created.

Reply:
Done